docs/core-concepts/app-deployment.md

```diff
@@ -217,37 +219,39 @@ Deploying a self-contained app with no third-party dependencies involves creatin
 Note that you also need to add a comma to separate the `runtimes` section from the previous section.
 A complete sample `project.json` file appears later in this section.
 
-5. Run the `dotnet restore` command to restore the dependencies specified in your project.
+6. Run the `dotnet restore` command to restore the dependencies specified in your project.
 
-6. Create debug builds of your app on each of the target platforms by using the `dotnet build -r` command and specifying the [runtime identifier](./rid-catalog.md) of the platform. In our example, we would run two separate builds as follows:
+7. Create debug builds of your app on each of the target platforms by using the `dotnet build -r` command and specifying the [runtime identifier](./rid-catalog.md) of the platform. In our example, we would run two separate builds as follows:
 
-    ```project.json
+    ```console
     dotnet build -r win10-x64
```

Comment:
I don't think you actually need to pass in a RID here.  I don't think the CLI is cross-compiling by RID since that isn't a feature that's supported yet (https://github.com/dotnet/cli/issues/465).  /cc @blackdwarf @piotrpMSFT